pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+name = "mammosunits"
+version = "0.1.0"
+description = "A short description of your project"
+authors = [{ name = "Sam Holt", email = "samjrholt@gmail.com" },
+           { name = "Hans Fangohr", email = "hans.fangohr@mpsd.mpg.de"} ]
+requires-python = ">=3.9"
+dependencies = [
+    "pytest",
+    "astropy>=7.0.0",
+    "jupyter",
+]
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["osx-arm64"]
+
+[tool.pixi.pypi-dependencies]
+mammosunits = { path = ".", editable = true }
```

Comment:
TODO: As @samjrholt mentioned, we need to figure out the problem with `pixi shell`. Maybe you can add a comment here Sam?